csds-client/README.md

```diff
@@ -0,0 +1,57 @@
+# CSDS Client
+[Client status discovery service (CSDS)](https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/status/v3/csds.proto) is a generic xDS API that can be used to get information about data plane clients from the control plane’s point of view. It is useful to enhance debuggability of the service mesh, where lots of xDS clients are connected to the control plane.<br/>
+The CSDS client is developed as a generic tool that can be used/extended to work with different xDS control planes.<br/>
+For now, this initial version of this CSDS client only support GCP's [Traffic Director](https://cloud.google.com/traffic-director).
+<br/>Before you start, you'll need [Go](https://golang.org/) installed.
+
+# Building
+* install dependencies using `go get`.
+
+# Running
+* run with `go run main.go <flag>`, e.g. <br/><br/>
+   * running with auto authentication mode, run with 
```

Comment:
I think we can just say: auto authentication mode:

same below.

Reply:
Solved.